recipes/itolapi/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "itolapi" %}
+{% set version = "4.0.0" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 161759384603664ff21ad6ae412a01549ce19436fc6959478d932d9b6c9915c4
+
+build:
+  noarch: generic
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
The recommended way is to use pip, as in
```
script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
```
Have you tried this?

Reply:
I'll have a go. I'll test locally and will commit if successful.  thanks

Reply:
same problem. I can keep the new simplified version using your code but I still need to restrict the build for python <3.8

Reply:
Thanks for changing this. It wasn’t intended as a solution to the failing build on Python 3.8, no idea about that.